src/test/e2e/setup/auth.setup.ts

```diff
@@ -23,7 +23,7 @@ setup('authenticate', async ({ baseURL, page }) => {
   // Wait for network to be idle, if we save storage too early, needed storage values might not yet be available
   await page.waitForLoadState('networkidle');
 
-  await page.getByText('Welkom, end to end').waitFor();
+  await page.getByText('Welkom').waitFor();
```

Comment:
Was it necessary to remove the name?
```suggestion
  await page.getByText('Welkom, end to end').waitFor();
```

Reply:
This was to make the tests more flexible to using different `E2E_TEST_` variables since currently you can configure them but the tests expect this very specific user